### PR TITLE
refactor(secnav): update secondary navigation component specs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`<hx-secnav>` :  implementing color palette 2.0 styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF2143

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by danielmdesigns — CHANGES_REQUESTED** [review]
For the dropdown section headings, we want to remove all caps. They should be title case, no more all caps.

File: docs/components/secnav/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
   {% endblock %}
 
   {% block content %}
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:21]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 2]
```diff
@@ -28,14 +28,14 @@ <h3>
       </h3>
       <br />
     </header>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:31]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 3]
```diff
@@ -53,17 +53,17 @@ <h3>
               <hx-icon type="bell"></hx-icon>
               <p>Notifications</p>
             </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:56]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 4]
```diff
@@ -53,17 +53,17 @@ <h3>
...
               <p>Notifications</p>
             </a>
-
+  
             <a href="#" class="selected">
               <hx-icon type="ticketing"></hx-icon>
               <p>Tickets</p>
             </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:61]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 5]
```diff
@@ -53,17 +53,17 @@ <h3>
...
               <p>Tickets</p>
             </a>
-
+  
             <a href="#">
               <hx-icon type="support"></hx-icon>
               <p>Support</p>
             </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:66]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 6]
```diff
@@ -72,15 +72,15 @@ <h3>
                 in your organization to request access to this item.
               </hx-tooltip>
             </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:75]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 7]
```diff
@@ -101,7 +101,7 @@ <h3>
           </div>
         </div>
       </nav>
-
+
```
Comment (100stacks) [inline docs/components/secnav/index.html:104]: Please remove the extra space.

File: docs/components/secnav/index.html [hunk 8]
```diff
@@ -28,14 +28,14 @@ <h3>
...
-
+  
     <div id="vue-secNavDemo">
       <nav id="hxTopNav">
         <a class="hxTopNavLogo" href="#">
           <img src="images/helix-logo.svg" alt="Brand Logo" />
         </a>
         <div class="hxTopNavMenu hxTopNavOptionMenu">
-          <hx-disclosure aria-controls="topnav-menu-options">
+          <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/index.html:38]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

File: docs/components/secnav/index.html [hunk 9]
```diff
@@ -72,15 +72,15 @@ <h3>
...
+  
             <a href="#">
               <hx-icon type="account"></hx-icon>
               <p>Account</p>
             </a>
           </div>
           <div class="hxSpacer"></div>
           <div class="hxTopNavMenu">
-            <hx-disclosure aria-controls="demo-user-menu">
+            <hx-disclosure aria-controls="demo-user-menu" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/index.html:83]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

File: docs/components/secnav/secnav-demo.js [hunk 10]
```diff
@@ -1,35 +1,35 @@
...
+      },
+      snippet: function () {
+        return Util.snippet(`
                 <nav id="hxTopNav">
                 <a class="hxTopNavLogo" href="#">
                   <img src="images/helix-logo.svg" alt="Brand Logo" />
                 </a>
                 <div class="hxTopNavMenu hxTopNavOptionMenu">
-                  <hx-disclosure aria-controls="topnav-menu-options">
+                  <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:32]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

File: docs/components/secnav/secnav-demo.js [hunk 11]
```diff
@@ -66,15 +66,15 @@ if (document.getElementById('vue-secNavDemo')) {
...
+          
                     <a href="#">
                       <hx-icon type="account"></hx-icon>
                       <p>Account</p>
                     </a>
                   </div>
                   <div class="hxSpacer"></div>
                   <div class="hxTopNavMenu">
-                    <hx-disclosure aria-controls="demo-user-menu">
+                    <hx-disclosure aria-controls="demo-user-menu" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:77]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

File: docs/components/secnav/secnav-demo.js [hunk 12]
```diff
@@ -95,7 +95,7 @@ if (document.getElementById('vue-secNavDemo')) {
                   </div>
                 </div>
               </nav>
-
+
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:98]: Please remove the extra tabs.

File: docs/components/secnav/secnav-demo.js [hunk 13]
```diff
@@ -66,15 +66,15 @@ if (document.getElementById('vue-secNavDemo')) {
                         in your organization to request access to this item.
                       </hx-tooltip>
                     </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:69]: Please remove the extra tabs.

File: docs/components/secnav/secnav-demo.js [hunk 14]
```diff
@@ -47,17 +47,17 @@ if (document.getElementById('vue-secNavDemo')) {
...
                       <p>Tickets</p>
                     </a>
-
+          
                     <a href="#">
                       <hx-icon type="support"></hx-icon>
                       <p>Support</p>
                     </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:60]: Please remove the extra tabs.

File: docs/components/secnav/secnav-demo.js [hunk 15]
```diff
@@ -47,17 +47,17 @@ if (document.getElementById('vue-secNavDemo')) {
...
                       <p>Notifications</p>
                     </a>
-
+          
                     <a href="#" class="selected">
                       <hx-icon type="ticketing"></hx-icon>
                       <p>Tickets</p>
                     </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:55]: Please remove the extra tabs.

File: docs/components/secnav/secnav-demo.js [hunk 16]
```diff
@@ -47,17 +47,17 @@ if (document.getElementById('vue-secNavDemo')) {
                       <hx-icon type="bell"></hx-icon>
                       <p>Notifications</p>
                     </a>
-
+
```
Comment (100stacks) [inline docs/components/secnav/secnav-demo.js:50]: Please remove the extra tabs.

File: src/scss/components/secnav/index.scss [hunk 17]
```diff
@@ -56,36 +53,35 @@
...
 
     hx-menuitem {
-      color: $gray-400;
+      color: $gray-950;
       padding: 0.5rem 1.25rem;
 
       &:hover {
-        background-color: $gray-800;
-        color: $gray-0 !important;
+        color: #970c8d !important;
```
Comment (100stacks) [inline src/scss/components/secnav/index.scss:81]: Please update to:

```css
color: $purple-500 !important;
```

File: src/scss/components/secnav/index.scss [hunk 18]
```diff
@@ -97,8 +93,8 @@
     }
 
     .hxDivider {
-      border-color: $gray-800;
-      height: 1px;
+      border-color: $gray-300;
+      height: 0.063rem;
```
Comment (100stacks) [inline src/scss/components/secnav/index.scss:97]: Please revert this change:

```css
height: 1px;
```

File: src/scss/components/secnav/index.scss [hunk 19]
```diff
@@ -56,36 +53,35 @@
       @include hxSecNavControl(disabled);
     }
 
+    &[aria-expanded="true"] {
+      color: #98008d;
```
Comment (100stacks) [inline src/scss/components/secnav/index.scss:57]: Please change to:

```css
color: $purple-500;
```

File: docs/components/secnav/test.html [hunk 20]
```diff
@@ -5,113 +5,113 @@
...
-          <hx-icon type="account"></hx-icon>
-          <p>Account</p>
-        </a>
-      </div>
-      <div class="hxSpacer"></div>
-      <div class="hxTopNavMenu">
-        <hx-disclosure aria-controls="demo-user-menu">
-          <span>Jane User</span>
+        <div class="hxTopNavMenu hxTopNavOptionMenu">
+          <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/test.html:13]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

File: docs/components/secnav/test.html [hunk 21]
```diff
@@ -5,113 +5,113 @@
...
+            </a>
+  
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </div>
+          <div class="hxSpacer"></div>
+          <div class="hxTopNavMenu">
+            <hx-disclosure aria-controls="demo-user-menu" aria-expanded="true">
```
Comment (100stacks) [inline docs/components/secnav/test.html:58]: Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

**Review by 100stacks — CHANGES_REQUESTED** [review]
### Please see changes below and squash commits.

**Comment by lalithkarikelli** [comment]
@danielmdesigns i have removed all caps for dropdown section headings . Please review the change. Thanks !

**Review by danielmdesigns — APPROVED** [review]
Looks good for first phase release!

File: docs/components/secnav/index.html (hunk 1 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:21]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 2 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html:31]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 8 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html]: Changes have been done, please review.

File: docs/components/secnav/index.html (hunk 9 above)
Comment (lalithkarikelli) [reply docs/components/secnav/index.html]: Changes have been done, please review.

File: docs/components/secnav/test.html (hunk 20 above)
Comment (lalithkarikelli) [reply docs/components/secnav/test.html]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss (hunk 19 above)
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss (hunk 18 above)
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: src/scss/components/secnav/index.scss (hunk 17 above)
Comment (lalithkarikelli) [reply src/scss/components/secnav/index.scss]: Changes have been done, please review.

File: docs/components/secnav/secnav-demo.js (hunk 11 above)
Comment (lalithkarikelli) [reply docs/components/secnav/secnav-demo.js]: Changes have been done, please review.

File: docs/components/secnav/secnav-demo.js (hunk 10 above)
Comment (lalithkarikelli) [reply docs/components/secnav/secnav-demo.js]: Changes have been done, please review.

File: docs/components/secnav/secnav-demo.js (hunk 12 above)
Comment (lalithkarikelli) [reply docs/components/secnav/secnav-demo.js]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM
